python/pyspark/sql/udf.py

```diff
@@ -310,14 +310,22 @@ def registerJavaFunction(self, name, javaClassName, returnType=None):
         ...     "javaStringLength", "test.org.apache.spark.sql.JavaStringLength", IntegerType())
```

Comment:
Ah, seems we need to fix `:param returnType:` across all other related APIs saying it takes DDL-formatted type string. 

@ueshin, mind opening a minor PR for this - `udf`, `pandas_udf`, `registerJavaFunction` and `register`  separately? If you are busy, will do it tonight. Doing this here is fine to me too, up to you.

Reply:
Sure, I'll update them here soon.